app/models/manageiq/providers/vmware/infra_manager/provision/configuration/network.rb

```diff
@@ -11,7 +11,7 @@ def build_config_network_adapters(vmcs)
       requested_networks.each_with_index do |net, idx|
         vim_net_adapter = template_networks[idx]
 
-        if net[:is_dvs] == true
+        if net[:is_dvs] || net[:is_opaque]
           build_config_spec_dvs(net, vim_net_adapter, vmcs)
```

Comment:
need a better name for `build_config_spec_dvs` which now handles also `opaque` networks. any suggestions?

Reply:
How about a new method?

```
if net[:is_dvs]
  build_config_spec_dvs
elsif net[:is_opaque]
  build_config_spec_opaque
else
  build_config_spec_vlan
end
```

Reply:
yeah, that was the original version I had. But the code are mostly identical